projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts

```diff
@@ -24,14 +24,21 @@ export function checkElement(tabElement: TabElement) {
     }
     case TabbingOrderTypes.GENERIC_INPUT: {
       cy.focused()
-        .get('input')
-        .should('be.visible');
+        .should('have.prop', 'tagName')
+        .should('eq', 'INPUT');
       break;
     }
     case TabbingOrderTypes.GENERIC_NG_SELECT: {
       cy.focused().should('have.attr', 'type', 'ng-select');
       return;
     }
+    case TabbingOrderTypes.CX_PRODUCT_VIEW: {
+      cy.focused()
+        .should('have.prop', 'tagName')
+        .should('eq', 'DIV');
+      cy.focused().should('have.class', 'cx-product-layout');
```

Comment:
can't we chain it to the previous `.should`s ?

Reply:
Yes we can. Done